src/utils/wallet.js

```diff
@@ -256,13 +397,45 @@ class Wallet {
         }
     }
 
-    async createNewAccount(accountId, fundingKey, fundingContract) {
+    /********************************
+    Check account was created before saving 10 retries, 1s each
+    ********************************/
+    async checkAccountAndSave(accountId, keyPair) {
```

Comment:
not sure what is this trying to do. Is it trying to workaround case when network is running too slow and retries are exceeded on near-api-js side?

Reply:
@mattlockyer ?

Reply:
Yes this was a placeholder when timeouts were an issue, will remove